src/main/kotlin/com/sparetimedevs/consonance/App.kt

```diff
@@ -19,29 +28,38 @@ import io.ktor.routing.Routing
 import io.ktor.server.engine.embeddedServer
 import io.ktor.server.netty.Netty
 import io.ktor.websocket.WebSockets
-import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import org.bson.types.ObjectId
+import org.koin.ktor.ext.inject
+import org.koin.standalone.StandAloneContext.startKoin
 
 fun Application.module() {
 
-    val logger = this.log
-    Dependency.load()
+    initaliseDependencies()
+
+    val dataInitialiser : DataInitialiser by inject()
+    launch(Dispatchers.Default) {
+        preloadData(dataInitialiser)
+
+    }
 
     install(DefaultHeaders)
     install(CallLogging)
     install(Compression)
     install(WebSockets)
+    install(Authentication) { jwt { customConfigure() } }
     install(Routing) {
-        api(
-            Dependency.scoreRepository,
-            Dependency.composerRepository,
-            Dependency.userRepository,
-            logger
-        )
+        authenticate {
+            userRoutes()
```

Comment:
Not sure if we eventually need all these operations to be authenticated. For example, I can imagine anyone should be able to get all the scores. But that only some (admins) can add or delete scores. For now, I'm fine with this approach.

Reply:
I agree. I just added those for the sake of example. I made the token expiry pretty liberal at an hour for now as well. We can discuss what we need at some point soon.